sr_moveit_hand_config/scripts/sr_moveit_hand_config/generate_hand_srdf.py

```diff
@@ -129,20 +129,27 @@
         else:
             OUTPUT_PATH = None
 
-        # Upload or output the input string on the correct param namespace or file
-        if OUTPUT_PATH is None:
+        if load:
             rospy.loginfo(" Loading SRDF on parameter server")
```

Comment:
Is there a reason for the space between the comment and quote? (I know you didn't add this :))